src/Streamly/Internal/Data/Fold.hs

```diff
@@ -425,6 +419,9 @@ _Fold1 step = mkAccum step_ Nothing' toMaybe
 drainBy ::  Monad m => (a -> m b) -> Fold m a ()
 drainBy f = lmapM f drain
 
+-- |
```

Comment:
Empty documentation? Intentional? Only for the `/Internal/` tag?

Reply:
No time for that. This is just internal, not even pre-release.